src/Common/src/Interop/Linux/procfs/Interop.ProcFsStat.cs

```diff
@@ -16,6 +16,7 @@ internal static partial class procfs
         internal const string RootPath = "/proc/";
         private const string ExeFileName = "/exe";
         private const string StatFileName = "/stat";
+        private const string StatusFileName = "/status";
```

Comment:
I know that several of the procfs files have a more human-friendly variant and a more machine-friendly variant.  It appears here we're parsing the more human-friendly one.  Is there no machine-friendly equivalent that would involve less parsing of strings and more structured output that would be cheaper to read?

Reply:
`status` is the human friendly version of `stat` and `statm` with some additional information:
 - `VmHWM` for `PeakWorkingSet64`
 - `VmData` for `PrivateMemorySize64`
 - `VmSwap` for `PagedMemorySize64` and `PagedSystemMemorySize64`
 - `VmPeak` for `PeakVirtualMemorySize64`

Reply:
> stat and statm with some additional information

Ah, so we could get some of the additional information from reading statm, but not all?  Assuming yes, ok, this makes sense.

Reply:
`statm` actually does not contain more information than what we already get in `stat` so that wouldn't be very usefyl for us.